apps/console/src/features/applications/components/forms/inbound-saml-form.tsx

```diff
@@ -907,39 +911,39 @@ export const InboundSAMLForm: FunctionComponent<InboundSAMLFormPropsInterface> =
-                                                    label: t("console:develop.features.applications.forms.inboundSAML" +
-                                                        ".sections.ssoProfile.fields.artifactBinding.label"),
-                                                    value: "enableSignatureValidationForArtifactBinding"
-                                                }
-                                            ]}
-                                            readOnly={readOnly}
-                                            disabled={!isArtifactBindingEnabled}
-                                            data-testid={`${testId}-artifact-binding-signature-validation-checkbox`}
-                                        />
-                                        <Hint disabled={!isArtifactBindingEnabled}>
-                                            {t("console:develop.features.applications.forms.inboundSAML.sections" +
-                                                ".ssoProfile.fields.artifactBinding.hint")}
-                                        </Hint>
-                                    </Grid.Column>
-                                </Grid.Row>
-                             )
-                             : null
+                                ? (
+                                    <Grid.Row columns={1}>
+                                        <Grid.Column mobile={16} tablet={16} computer={16}>
+                                            <Field
+                                                ref={signatureValidationForArtifactBinding}
+                                                name="signatureValidationForArtifactBinding"
+                                                label=""
+                                                required={false}
+                                                requiredErrorMessage="this is needed"
+                                                type="checkbox"
+                                                value={
+                                                    isArtifactBindingEnabled && initialValues?.singleSignOnProfile
+                                                        .enableSignatureValidationForArtifactBinding ?
+                                                        ["enableSignatureValidationForArtifactBinding"] : []}
+                                                children={[
+                                                    {
+                                                        label: t("console:develop.features.applications.forms.inboundSAML" +
+                                                            ".sections.ssoProfile.fields.artifactBinding.label"),
+                                                        value: "enableSignatureValidationForArtifactBinding"
+                                                    }
+                                                ]}
+                                                readOnly={readOnly}
```

Comment:
```suggestion
                                                readOnly={ readOnly }
```
Check other places as well.

Reply:
Fixed with 9b8bdce8d7b4a57d300b33d30c62e351149c4a51